instead of using a custom findquazip with hardcoded versions,
just use the provided system cmake file
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6685ea4..13a1483 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,8 +51,8 @@ find_package(Qt5Concurrent REQUIRED)
 
 find_package(ZLIB REQUIRED)
 
-# find_package(Intl REQUIRED)
-find_package(QUAZIP REQUIRED)
+find_package(Intl REQUIRED)
+find_package(QuaZip-Qt5 REQUIRED)
 
 if(CLICK_MODE)
     # Find out the architecture for package building
@@ -111,7 +111,7 @@ add_executable(sturmreader
     ui/reader/comicreader.cpp
     ui/resources.qrc
     ${QML_COMPILED_RES})
-target_include_directories(sturmreader SYSTEM PRIVATE ${QUAZIP_INCLUDE_DIR})
+target_include_directories(sturmreader SYSTEM PRIVATE QuaZip::QuaZip)
 
 add_library(qtrarobjslib STATIC IMPORTED)
 set_property(TARGET qtrarobjslib PROPERTY IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/libqtrarobjs.a)
@@ -122,7 +122,7 @@ set_property(TARGET qtrarlib PROPERTY IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_D
 target_link_libraries(sturmreader
     poppler-qt5
     ${Intl_LIBRARIES}
-    ${QUAZIP_LIBRARIES}
+    QuaZip::QuaZip
     qtrarlib qtrarobjslib
     pthread
     ZLIB::ZLIB
